layout.lua

```diff
@@ -51,6 +51,12 @@ function handle_layout(args)
 	return retval
 end
 
+-- This returns the metadata for the current layout. Currently only `name` is
+-- supported, the name of the layout
+function handle_metadata(args)
```

Comment:
Please mention that the argument to this function is identical to that of `handle_layout`.

Reply:
And that this function is optional